src/main/kotlin/uk/gov/dwp/dataworks/controllers/UserContainerController.kt

```diff
@@ -33,21 +36,23 @@ class UserContainerController {
         ApiResponse(responseCode = "400", description = "Failure, bad request")
     ])
     @PostMapping("/deployusercontainers")
-    fun launchTask(@RequestBody requestBody: Model){
-        if (existingUserServiceCheck.check(requestBody.userName, requestBody.ecsClusterName)){
+    fun launchTask(@RequestBody requestBody: Model): String{
+        if (requestBody.userName!=null){
+        if (existingUserServiceCheck.check(requestBody.userName, configService.getStringConfig(ConfigKey.ECS_CLUSTER_NAME))){
             logger.info("Redirecting user to running containers, as they exist")
-            return
+        } else {
+            taskDeploymentService.taskDefinitionWithOverride(
+                    requestBody.userName,
+                    requestBody.containerPort,
```

Comment:
I believe this should also be set via environment variables as it is going to be static (probably 8443)